src/string_pool.cr

```diff
@@ -26,8 +26,10 @@ class StringPool
 
   # Creates a new empty string pool.
   def initialize
-    @buckets = Array(Array(String)?).new(11, nil)
+    @hashes = Pointer(Hash::Hasher::Value).malloc(8, 0_u32)
```

Comment:
This should use `@capacity` instead of fixed `8`.

Reply:
will do